src/api/app/models/event_subscription/for_role_form.rb

```diff
@@ -0,0 +1,53 @@
+class EventSubscription
+  class ForRoleForm
+    attr_reader :name, :channels, :subscriber
+
+    def initialize(role_name, event, subscriber)
+      @subscriber = subscriber
+      @name = role_name
+      @event = event
+      @channels = []
+    end
+
+    def call
+      @channels = EventSubscription.channels.keys.reject { |channel| channel == 'disabled' }.map do |channel|
```

Comment:
:thought_balloon: To** maket it shorter, you can extract this into a `EventSubscription#channels_without_disabled` :stuck_out_tongue:

Reply:
right!, thanks :smile_cat: